frontend/src/components/Table.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { MdModeEdit,MdDelete } from "react-icons/md";
+
+function Table({ products = [] }) {
+    return (
+        <div>
+            <table className="min-w-full shadow-md overflow-hidden">
+                <colgroup>
+                    <col className="w-12" />
+                    <col className="w-64" />
+                    <col className="w-15" />
+                    <col className="w-15" />
+                    <col className="w-15" />
+                </colgroup>
+                <thead>
+                    <tr className="bg-gray-400">
+                        <th className="px-4 py-2 text-white border border-gray-300 text-left">PID</th>
+                        <th className="px-4 py-2 text-white border border-gray-300 text-left">Product Name</th>
+                        <th className="px-4 py-2 text-white border border-gray-300 text-center">Price</th>
```

Comment:
Maybe add (THB) -> Price (THB), wdyt? Anyway, which means we support Thai Baht only.